src/components/commons/Stying/StackChildren/StackChildren.tsx

```diff
@@ -0,0 +1,25 @@
+import { PropsWithChildren } from 'react';
+
+type StackChildrenProps = {
+  direction: 'column' | 'column-reverse' | 'row' | 'row-reverse' | undefined;
+  gap: 'px' | 'em' | '%' | `calc(${string})`;
+};
+
+export const StackChildren = ({
+  children,
+  direction,
```

Comment:
```suggestion
  flexDirection = 'column',
```

どっちをデフォルトにするかはお任せします

Reply:
`column` でいきます